src/styles/blocks/card.scss

```diff
@@ -0,0 +1,97 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  box-sizing: border-box;
+  width: 200px;
+  background-color: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  margin: 0 24px;
```

Comment:
instead of margin for block it would be better to add `gap: 48px` style to `.main`
```suggestion
```

Reply:
I do not agree with this proposal, is it possible to describe this idea in more detail, or a link to the material with an example?

Reply:
fix, redid